static/js/rename.js

```diff
@@ -44,12 +41,10 @@ function rename() {
   updateButton.disabled = true;
   updateButton.innerHTML = "Renaming";
 
-  var data = new FormData();
-  data.append('name', document.getElementById("new-name").value);
-  data.append('group', document.getElementById("new-group").value);
+  var data = $("#rename-form").serialize()
 
   var xmlHttp = new XMLHttpRequest();
-  xmlHttp.open('POST', '/api/rename', true);
+  xmlHttp.open('POST', '/api/rename?'+data, true);
```

Comment:
Shouldn't the form data still be getting posted in the request body, not the args?

Reply:
Yeah I was having issues with putting it in the request body but I just forgot to set the `Contet-Type` to `application/x-www-form-urlencoded; charset=UTF-8`